components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$mah-cover-basic: 676px !default;
+$fz-cover-basic-button: 16px !default;
```

Comment:
I would rather to leave it here because right now it is too component-especific and reusability is not clear at all.